Unosquare.CodeChallenge/WarehouseTestingUnit/WarehouseServices/WarehouseServices_unitTest.cs

```diff
@@ -0,0 +1,145 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tests;
+using WarehouseCoreLib.DataAccess;
+using WarehouseModels.Models;
+using WarehouseServices.Services;
+using Assert = NUnit.Framework.Assert;
+using Theory = NUnit.Framework.TheoryAttribute;
+
+namespace WarehouseTestingUnit.WarehouseServices
+{
+    public class WarehouseServices_unitTest
+    {
+
+        [Fact]
+        public void Test_getAllCompanies_getAllCompaniesItems()
```

Comment:
UnitTest methods, same thing, PascalCase

Reply:
ok .. I'll make the changes

Reply:
I change the names of the test on the commit f4f6e2d1a270fce4c21d7acf8db531a02cde94fe removing the Test_